web/js/link/wv.link.ui.js

```diff
@@ -68,6 +68,54 @@ wv.link.ui = wv.link.ui || function(models, config) {
                 "</div>";
         }
         item += "</div>";
+
```

Comment:
Sure, I would be interested in doing that. I think this bit of functionality is perfect for taking a dive into react.